exporter/debugexporter/README.md

```diff
@@ -107,6 +108,38 @@ Attributes:
         {"kind": "exporter", "data_type": "traces", "name": "debug"}
 ```
 
+## Using the collector's internal logger
+
+When `use_internal_logger` is set to `true` (the default), the exporter uses the collector's [internal logger][internal_telemetry] for output.
+This comes with several consequences:
+
+1. The exporter's output is prefixed and suffixed by the internal logger's data.
+   - The prefix is the current timestamp and the `info` level marker, similar to `2023-11-10T22:49:03.510-0600    info`
+   - The suffix is the exporter's metadata in JSON format, similar to `{"kind": "exporter", "data_type": "traces", "name": "debug"}`
+1. The exporter's output is sent to the same destination as the collector's logs. By default, it is the [standard error][stderr] stream,
+   configurable via the [service.telemetry.logs.output_paths][configure_internal_logs] option.
```

Comment:
Just an FYI: we recently had a conversation about how best to notate nested keys in YAML paths. The ultimate decision was to use double colons (::) to avoid any confusion with namespaces using dot notation. 

See [this issue comment](https://github.com/open-telemetry/opentelemetry.io/issues/4484#issuecomment-2117608625) for details and [this tip](https://opentelemetry.io/docs/collector/configuration/#location) that was just added to the docs.

Thanks!

Reply:
Oh, right, makes total sense. Will update the docs. Thank you for pointing this out!

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-collector/pull/10227/commits/4f8f733841687a7e97917213df9ebdaa49b64684.